### PR TITLE
[0.2] Add tags to orders

### PR DESCRIPTION
**Pull request by alecritson**
This PR looks to add tagging to Orders and allow them to be searchable. @markmead Might ask you to take a peak at the UI and then once all good I can get them onto Products.

### CODE REVIEW

**Review by markmead — APPROVED** [review]
Looks good to me! 🥳

File: packages/admin/resources/views/livewire/components/orders/show.blade.php [hunk 1]
```diff
@@ -101,6 +101,18 @@
...
+            <header>
+                <strong class="text-gray-700">
+                  {{ __('adminhub::components.orders.show.tags_header') }}
+                </strong>
+            </header>
+            @livewire('hub.components.tags', [
+              'taggable' => $order,
+              'independant' => true,
+            ])
+            {{-- <x-hub::input.tags id="tags"/> --}}
```
Comment (glennjacobs) [inline packages/admin/resources/views/livewire/components/orders/show.blade.php:114]: Commented code to be removed?

**Comment by alecritson** [comment]
@glennjacobs Added the applied tags to the listing screen as a comma separated list.

**Comment by glennjacobs** [comment]
Need to show tags in the data table, as we have a filter we need to visibly be able to see them also.

**Comment by glennjacobs** [comment]
> @glennjacobs Added the applied tags to the listing screen as a comma separated list.

Thanks. Could we make them show as typical tag "pills"? Maybe something Mark could help wth.

**Comment by alecritson** [comment]
@glennjacobs Changed the design a bit so they look more like pills

**Review by glennjacobs — APPROVED** [review]

**Pull request merged**